src/configs/redoc/customer-accounts-api/specv2.yml

```diff
@@ -0,0 +1,228 @@
+openapi: 3.0.0
+info:
+  title: Customer Accounts API
+  description: >-
+    Customer Accounts API is responsible for the authentication and authorization processes of customers in LiveChat.
+  version: 2.0.0
+servers:
+  - url: 'https://accounts.livechat.com/v2/customer'
+    description: Main production server URL
+paths:
+  '{organization_id}/{group_id}/token/':
+    post:
+      summary: Create a token
+      description: Creates a new customer access token for a group in organization provided as parameters.
```

Comment:
perhaps, we should stick to the form: "create sth" instead of 3rd form "creates sth" - I know that this inconsistency exists in v1, too

Reply:
Also, I would emphasize that this is a very specific use-case and results in a special token that is intended to separate "identities" per group in an organization as opposed to the normal token that shares identities across groups in organization

Reply:
```suggestion
      description: Creates a new customer access token for a different group within the same organization. Use this token to separate customer identity across groups within one organization.
```

Reply:
We could also provide the link to https://github.com/livechat/livechat-public-docs/pull/1030/files#diff-a3066d38de12a7211d5e330219a37004ea3935bff79708752538b1ae513f1861R797

Reply:
As for create vs creates, I would stick to the 3rd form simply because this is what we have in other APIs. Inconsistencies from Customer SSO v1.0.0 should be fixed

Reply:
Ok, I'm sticking to the 3rd person then